pkg/mosn/mosn.go

```diff
@@ -171,6 +172,19 @@ func (m *Mosn) initializeMetrics() {
 		}
 		log.StartLogger.Infof("[mosn] [init metrics] create metrics sink: %v", cfg.Type)
 	}
+
+	// set ewma alpha
+	if config.EWMAConfig != nil {
+		switch {
+		case config.EWMAConfig.Alpha > 0 && config.EWMAConfig.Alpha < 1:
+			cluster.SetAlpha(config.EWMAConfig.Alpha)
+		case config.EWMAConfig.Target > 0 && config.EWMAConfig.Target < 1 && config.EWMAConfig.Duration != nil:
+			cluster.SetAlpha(ewma.Alpha(config.EWMAConfig.Alpha, config.EWMAConfig.Duration.Duration))
```

Comment:
When I was performing Chaos Tests, I found that result was not matching expectation. 

I'm sorry, I found this is a mistake